### DISCUSSION TITLE
AntiSpam unban

### DISCUSSION THREAD

**Discussion by m-Phoenix852** [GriefPrevention/GriefPrevention#1209] [Q&A]
### What happened:
_Player banned for spamming, can't unban now_

### What was expected:
_The player was banned for spamming, I don't see any command to unban him. I tried unban, unban-ip, pardon, pardon-ip. None works. Please help._

**Comment by TDMidkwhatisxd** [comment]
delete banned file xd

**Comment by RoboMWM** [comment]
you're missing a ton of info that you deleted, but I guess your ban plugin is hijacking all of those commands. Try `/minecraft:pardon`.

**Reply by m-Phoenix852** [reply]
> you're missing a ton of info that you deleted, but I guess your ban plugin is hijacking all of those commands. Try `/minecraft:pardon`.

tried man, nothing works

**Reply by RoboMWM** [reply]
GP only uses the vanilla ban system by default. Unless you changed how banning works in the config, that's the only place it exists.

**Comment by RoboMWM** [comment]
Also, this is a question not a bug report. Moving to discussions.
